src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -44,4 +54,27 @@ protected Employee toEntity(EmployeeDto entity) {
         employee.setRoles(roles);
         return employee;
     }
+
+    @Override
+    @Transactional
+    public EmployeeDto create(EmployeeDto employeeDto){
+        Employee entityToCreate = toEntity(employeeDto);
+        Employee savedEntity = getRepository().save(entityToCreate);
+        Set<PaymentInformation> paymentInformationSet = paymentInformationService.create(employeeDto.paymentInformation(),savedEntity);
+        savedEntity.setPaymentsInformation(paymentInformationSet);
```

Comment:
You could remove this line, because we set the payment information in the toDto method